library_generation/generate_pr_description.py

```diff
@@ -158,7 +158,7 @@ def __filter_qualified_commit(paths: Dict[str, str], commit: Commit) -> (Commit,
     """
     for file in commit.stats.files.keys():
         versioned_proto_path = find_versioned_proto_path(file)
-        if versioned_proto_path in paths:
+        if versioned_proto_path in paths and (not file.endswith("BUILD.bazel")):
```

Comment:
Do we have a unit test for this case? Or it is tested indirectly in the golden test?

Reply:
It's tested in the golden test.

I changed the commit in the test configuration file and `chore(ruby)` didn't make it in the golden file.

Reply:
OK, it would be great if we can add unit tests for this class in the future so that don't have to update the goldens for un-related changes, but it's fine for now.